src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/Ticketing_TicketTabNavStack/TicketTabNav_PurchaseTabScreen/types.ts

```diff
@@ -14,7 +14,7 @@ export type RecentFareContract = {
   preassignedFareProduct: PreassignedFareProduct;
   fromTariffZone?: TariffZone;
   toTariffZone?: TariffZone;
-  orderId: string;
+  direction: string;
```

Comment:
Since this is the mapped type, would it be appropriate to use `TravelRightDirection` instead of `string` here?

Reply:
Fixed now :)